documentation/index.html.md

```diff
@@ -22,19 +22,24 @@ If you are comfortable with JavaScript, the epicenter.js library is an easy way
 <a name="concepts"></a>
 ###Concepts in Epicenter.js
 
-The epicenter.js library is a set of services (adapters) and managers to help streamline your work with the underlying Epicenter APIs.
+The epicenter.js library is a set of services (adapters) and managers to help streamline your work with the underlying Epicenter APIs. Typically, you'll develop using the managers:
 
-* The following services are direct adaptations of the underlying [RESTful APIs](../rest_apis/): 
-	* [Run Service](./generated/run-api-service/)
-	* [Data Service](./generated/data-api-service/)
-	* [Auth Service](./generated/auth-api-service)
-	* [Variables Service](./generated/variables-api-service/)
-	* [World Adapter](./generated/world-api-adapter/)
+* [Authorization Manager](./generated/auth-manager/): The Authorization Manager provides an easy way to manage user authentication (logging in and out) and authorization (keeping track of tokens, sessions, and groups) for projects.
+* [Epicenter Channel Manager](./generated/epicenter-channel-manager/) and the underlying [Channel Manager](./generated/channel-manager/): Once you've created "[worlds](../glossary/#world)" with the World Manager, you often want to end users in each world to be able to communicate: to talk with each other (e.g. a user-to-user chat feature), or to receive updates when something changes in their world (e.g. variables associated with their shared run are updated). The Epicenter Channel Manager provides a publish/subscribe channel using cometd.
```

Comment:
if we're going to break alphabetical, then i think it should be: 
Auth, Run, World, Channel. sound ok?